CRM/Upgrade/Incremental/sql/5.20.alpha1.mysql.tpl

```diff
@@ -1 +1,5 @@
 {* file to handle db changes in 5.20.alpha1 during upgrade *}
+
+-- CRM-21740
+ALTER TABLE  civicrm_relationship_type ADD weight INT NOT NULL;
```

Comment:
@yashodha we have generally used php addColumn function in the upgrade of late as its more upgrade safe

Reply:
@seamuslee001 I have made the proposed changes.